src/tools/clippy/clippy_utils/src/hir_utils.rs

```diff
@@ -701,8 +707,8 @@ impl<'a, 'tcx> SpanlessHash<'a, 'tcx> {
                 }
             },
             ExprKind::If(ref cond, ref then, ref else_opt) => {
-                let c: fn(_, _, _) -> _ = ExprKind::If;
-                c.hash(&mut self.s);
+                //let c: fn(_, _, _) -> _ = ExprKind::If;
+                //c.hash(&mut self.s);
```

Comment:
Why can this be removed? And if it can, please remove it completely instead of commenting it out.